.eslintignore

```diff
@@ -4,4 +4,5 @@ speech/
 training_data/
 ui/ibm/watson-speech.min.js
 ui_config/
-/*.js
```

Comment:
I didn't edit that line. I just added the libs folder below it because that where I put the snap svg library with modification.